src/js/common/polyfills.js

```diff
@@ -24,3 +24,13 @@ if (!Modernizr.fetch) {
 
 // This polyfill has its own test logic so no need to conditionally require.
 require('polyfill-function-prototype-bind');
+
+// Disable scrolling on input[type="number"] because it was changing the value unexpectedly
+// NOTE: This also just stops the user from scrolling when hovering over a number
+//  input that has focus. This isn't ideal, but until a solution is found to that,
+//  this is what we've got.
+document.addEventListener('mousewheel', (event) => {
```

Comment:
@cehsu Oh, and yes, we do need that `document.activeElement === event.target`, otherwise it'll stop scrolling on _all_ number fields whether or not they have the focus. The problem with the scrolling affecting the value in the number field only happens when the field has focus and we're hovering over it, so this takes care of that. Just narrows the weird UX is all. 😛